src/accounts/Person.java

```diff
@@ -0,0 +1,21 @@
+package accounts;
+
+/**
+ *
+ * @author aswecker
+ */
+public interface Person {
+    
+    public String Getfname();
+    public void Setfname(String infname);
+    public String Getlname();
+    public void Setlname(String inlname);
+    public int Getuserid();
+    public void Setuserid(int inuserid);   
```

Comment:
Are you sure we need this? Changing the account ID after creating the account will require updating all the saved data with the new ID. It might be better to make that field final and not let it be changed.